catalogs/catalog-lakehouse-hudi/build.gradle.kts

```diff
@@ -115,12 +110,29 @@ dependencies {
   testImplementation(libs.hadoop2.auth) {
     exclude("*")
   }
+  testImplementation(libs.hadoop2.hdfs)
   testImplementation(libs.hadoop2.mapreduce.client.core) {
     exclude("*")
   }
   testImplementation(libs.htrace.core4)
   testImplementation(libs.junit.jupiter.api)
-  testImplementation(libs.woodstox.core)
+  testImplementation(libs.mysql.driver)
+  testImplementation(libs.postgresql.driver)
+  testImplementation(libs.prometheus.dropwizard)
```

Comment:
Why do we need these jars for test?

Reply:
`libs.mysql.driver` and `libs.postgresql.driver` are for Gravitino different backends IT.

For `libs.prometheus.dropwizard`, if I removed it, will encountered the below error:
```
Caused by: java.lang.NoSuchMethodError: io.prometheus.client.dropwizard.DropwizardExports.<init>(Lcom/codahale/metrics/MetricRegistry;Lio/prometheus/client/dropwizard/samplebuilder/SampleBuilder;)V
	at org.apache.gravitino.metrics.MetricsSystem.registerMetricsToPrometheusRegistry(MetricsSystem.java:186)
	at org.apache.gravitino.metrics.MetricsSystem.start(MetricsSystem.java:121)
	at org.apache.gravitino.GravitinoEnv.start(GravitinoEnv.java:312)
	at org.apache.gravitino.server.GravitinoServer.start(GravitinoServer.java:147)
	at org.apache.gravitino.server.GravitinoServer.main(GravitinoServer.java:169)
	at org.apache.gravitino.integration.test.MiniGravitino.lambda$start$0(MiniGravitino.java:150)
```
Because the `hudi_spark_bundle` jar includes the same class `DropwizardExports` which is conflict with the Gravitino's dependency so I have to declare the `libs.prometheus.dropwizard` dependency explicitly